iOS 1.0 installation (Carthage or xcodebuild archive) failures

## Bug report

Cannot generate XCFramework for apollo-ios 1.0.0 using either Carthage or building manually. 

## Versions

Please fill in the versions you're currently using: 

- `apollo-ios` SDK version: 1.0.0
- Xcode version: 13.4.1
- Swift version: N/A
- Package manager: Carthage or manually.

## Steps to reproduce

When using Carthage, the steps are:
```
carthage update --use-xcframeworks --platform ios
*** Fetching apollo-ios
*** Checking out apollo-ios at "1.0.0"
*** xcodebuild output can be found in /var/folders/18/y_v7x80j4v5dyfx8lxfzx1mr0000gq/T/carthage-xcodebuild.I6I81s.log
*** Building scheme "ApolloAPI" in Apollo.xcodeproj
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -project /Users/user/Documents/carthage-project/Carthage/Checkouts/apollo-ios/Apollo.xcodeproj -scheme ApolloAPI -configuration Release -derivedDataPath /Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive VALIDATE_WORKSPACE=NO -archivePath /var/folders/18/y_v7x80j4v5dyfx8lxfzx1mr0000gq/T/apollo-ios SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/user/Documents/carthage-project/Carthage/Checkouts/apollo-ios)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/18/y_v7x80j4v5dyfx8lxfzx1mr0000gq/T/carthage-xcodebuild.I6I81s.log
user@R6TJ4Y9F9W carthage-project % open /var/folders/18/y_v7x80j4v5dyfx8lxfzx1mr0000gq/T/carthage-xcodebuild.I6I81s.log
```

Log file:
```
note: Using new build system
note: Using codesigning identity override: 
note: Planning
Analyze workspace

Create build description
Build description signature: 146c2f8adbbd7f0a6aae0cbffa5f47b2
Build description path: /Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0/Build/Intermediates.noindex/ArchiveIntermediates/ApolloAPI/IntermediateBuildFilesPath/XCBuildData/146c2f8adbbd7f0a6aae0cbffa5f47b2-desc.xcbuild

note: Build preparation complete
note: Building targets in dependency order
/Users/user/Documents/cartage-project/Carthage/Checkouts/apollo-ios/Configuration/Apollo/Apollo-Project-Release.xcconfig:1:1: error: unable to open file (in project "Apollo")
/Users/user/Documents/cartage-project/Carthage/Checkouts/apollo-ios/Configuration/Apollo/Apollo-Target-ApolloAPI.xcconfig:1:1: error: unable to open file (in target "ApolloAPI" in project "Apollo")
/Users/user/Documents/cartage-project/Carthage/Checkouts/apollo-ios/Configuration/Apollo/Apollo-Project-Release.xcconfig:1:1: error: unable to open file (in project "Apollo")
/Users/user/Documents/cartage-project/Carthage/Checkouts/apollo-ios/Configuration/Apollo/Apollo-Target-ApolloAPI.xcconfig:1:1: error: unable to open file (in target "ApolloAPI" in project "Apollo")
/Users/user/Documents/cartage-project/Carthage/Checkouts/apollo-ios/Configuration/Apollo/Apollo-Project-Release.xcconfig:1:1: error: unable to open file (in project "Apollo")
/Users/user/Documents/cartage-project/Carthage/Checkouts/apollo-ios/Configuration/Apollo/Apollo-Target-ApolloAPI.xcconfig:1:1: error: unable to open file (in target "ApolloAPI" in project "Apollo")
error: Multiple commands produce '/Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0/Build/Intermediates.noindex/ArchiveIntermediates/ApolloAPI/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/.framework'
    note: Target 'ApolloAPI' (project 'Apollo') has link command with output '/Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0/Build/Intermediates.noindex/ArchiveIntermediates/ApolloAPI/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/.framework'
    note: Target 'ApolloAPI' (project 'Apollo') has create directory command with output '/Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0/Build/Intermediates.noindex/ArchiveIntermediates/ApolloAPI/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/.framework'
warning: Traditional headermap style is no longer supported; please migrate to using separate headermaps and set 'ALWAYS_SEARCH_USER_PATHS' to NO. (in target 'ApolloAPI' from project 'Apollo')
error: Value for SWIFT_VERSION cannot be empty. (in target 'ApolloAPI' from project 'Apollo')
warning: duplicate output file '/Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0/Build/Intermediates.noindex/ArchiveIntermediates/ApolloAPI/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/.framework' on task: MkDir /Users/user/Library/Caches/org.carthage.CarthageKit/DerivedData/13.4.1_13F100/apollo-ios/1.0.0/Build/Intermediates.noindex/ArchiveIntermediates/ApolloAPI/IntermediateBuildFilesPath/UninstalledProducts/iphoneos/.framework (in target 'ApolloAPI' from project 'Apollo')
** ARCHIVE FAILED **
```

When building manually:
```
xcodebuild archive \
		-configuration Release \
		-project "Apollo.xcodeproj" \
		-scheme "Apollo" \
		SKIP_INSTALL=NO \
		-archivePath "./build/Apollo.framework-iphoneos.xcarchive" \
		-destination generic/platform=iOS \
	    BUILD_LIBRARY_FOR_DISTRIBUTION=YES

/Users/user/Documents/apollo-ios/Sources/ApolloAPI/Selection+Conditions.swift:37:16: note: 'inverted' declared here
    public let inverted: Bool
               ^
/Users/user/Documents/apollo-ios/Sources/ApolloAPI/Selection+Conditions.swift:48:12: error: 'let' property 'variableName' may not be initialized directly; use "self.init(...)" or "self = ..." instead
      self.variableName = value
           ^
/Users/user/Documents/apollo-ios/Sources/ApolloAPI/Selection+Conditions.swift:36:16: note: 'variableName' declared here
    public let variableName: String
               ^
/Users/user/Documents/apollo-ios/Sources/ApolloAPI/Selection+Conditions.swift:49:12: error: 'let' property 'inverted' may not be initialized directly; use "self.init(...)" or "self = ..." instead
      self.inverted = false
           ^
/Users/user/Documents/apollo-ios/Sources/ApolloAPI/Selection+Conditions.swift:37:16: note: 'inverted' declared here
    public let inverted: Bool
               ^

** ARCHIVE FAILED **


The following build commands failed:
	CompileSwift normal arm64 (in target 'ApolloAPI' from project 'Apollo')
	CompileSwiftSources normal arm64 com.apple.xcode.tools.swift.compiler (in target 'ApolloAPI' from project 'Apollo')
(2 failures)
```

Thank you for your time!

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -146,6 +146,21 @@ jobs:
       - run:
           command: swift build
 
+  XCFramework_Build:
+    macos:
+      xcode: << pipeline.parameters.xcode_version >>
+    steps:
+      - common_test_setup
+      - run:
+          name: Build XCArchive (iOS Simulator)
+          command: xcodebuild archive -configuration Release -project "Apollo.xcodeproj" -scheme "Apollo" -destination 'generic/platform=iOS Simulator' -archivePath "./build/iphonesimulator.xcarchive" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES
+      - run:
+          name: Build XCArchive (iOS)
+          command: xcodebuild archive -configuration Release -project "Apollo.xcodeproj" -scheme "Apollo" -destination 'generic/platform=iOS' -archivePath "./build/iphoneos.xcarchive" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES
+      - run:
+          name: Build XCFramework
+          command: xcodebuild -create-xcframework -output ./build/Apollo.xcframework -framework ./build/iphonesimulator.xcarchive/Products/@rpath/Apollo.framework -framework ./build/iphoneos.xcarchive/Products/@rpath/Apollo.framework
+
   IntegrationTests_macOS_current:
     macos:
       xcode: << pipeline.parameters.xcode_version >>
@@ -242,6 +257,8 @@ workflows:
     jobs:
       - Swift_Build:
           name: Build as Swift package
+      - XCFramework_Build:
+          name: Build XCFramework
       - IntegrationTests_macOS_current:
           name: Apollo Integration Tests - macOS << pipeline.parameters.macos_version >>
       - macOS_current:

--- a/Sources/ApolloAPI/CacheKeyInfo.swift
+++ b/Sources/ApolloAPI/CacheKeyInfo.swift
@@ -111,7 +111,7 @@ public struct CacheKeyInfo {
   ///   - id: The unique cache key for the response object for the ``CacheKeyInfo``.
   ///   - uniqueKeyGroup: An optional identifier for a group of objects that should be grouped
   ///     together in the `NormalizedCache`.
-  @inlinable public init(id: String, uniqueKeyGroup: String? = nil) {
+  public init(id: String, uniqueKeyGroup: String? = nil) {
     self.id = id
     self.uniqueKeyGroup = uniqueKeyGroup
   }

--- a/Sources/ApolloAPI/CacheReference.swift
+++ b/Sources/ApolloAPI/CacheReference.swift
@@ -61,7 +61,7 @@ public struct CacheReference: Hashable {
   ///
   /// - Parameters:
   ///   - key: The unique identifier for the referenced object.
-  @inlinable public init(_ key: String) {
+  public init(_ key: String) {
     self.key = key
   }
 

--- a/Sources/ApolloAPI/DataDict.swift
+++ b/Sources/ApolloAPI/DataDict.swift
@@ -4,7 +4,7 @@ public struct DataDict: Hashable {
   public var _data: JSONObject
   public let _variables: GraphQLOperation.Variables?
 
-  @inlinable public init(
+  public init(
     _ data: JSONObject,
     variables: GraphQLOperation.Variables?
   ) {

--- a/Sources/ApolloAPI/Selection+Conditions.swift
+++ b/Sources/ApolloAPI/Selection+Conditions.swift
@@ -9,15 +9,15 @@ public extension Selection {
   struct Conditions: Hashable {
     public let value: [[Condition]]
 
-    @inlinable public init(_ value: [[Condition]]) {
+    public init(_ value: [[Condition]]) {
       self.value = value
     }
 
-    @inlinable public init(_ conditions: [Condition]...) {
+    public init(_ conditions: [Condition]...) {
       self.value = Array(conditions)
     }
 
-    @inlinable public init(_ condition: Condition) {
+    public init(_ condition: Condition) {
       self.value = [[condition]]
     }
 
@@ -36,7 +36,7 @@ public extension Selection {
     public let variableName: String
     public let inverted: Bool
 
-    @inlinable public init(
+    public init(
       variableName: String,
       inverted: Bool
     ) {
@@ -44,7 +44,7 @@ public extension Selection {
       self.inverted = inverted;
     }
 
-    @inlinable public init(stringLiteral value: StringLiteralType) {
+    public init(stringLiteral value: StringLiteralType) {
       self.variableName = value
       self.inverted = false
     }

--- a/Sources/ApolloAPI/Selection.swift
+++ b/Sources/ApolloAPI/Selection.swift
@@ -20,7 +20,7 @@ public enum Selection {
       return alias ?? name
     }
 
-    @inlinable public init(
+    public init(
       _ name: String,
       alias: String? = nil,
       type: OutputType,

